moj-node/server/views/components/header/template.njk

```diff
@@ -2,17 +2,16 @@
   <div class="govuk-hub-header__container govuk-width-container govuk-!-padding-0">
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-one-half">
-        {% if params.user %}
+        {% if false %}
           <div id="hub-auth-header" class="hub-header-user-container">
             <span class="govuk-!-font-weight-bold">{{ params.user.name }}</span>
             <h1>{{ params.appName }}</h1>
           </div>
-        {% else %}
-          <a href="/" class="govuk-hub-header__link govuk-hub-header__link--homepage">
-            <span class="govuk-hub-header__title govuk-body govuk-!-font-weight-bold">{{ params.appName }}</span>
-          </a>
         {% endif %}
 
+        <a href="/" class="govuk-hub-header__link govuk-hub-header__link--homepage">
```

Comment:
DId this need to move? It'll always execute where it was when doing if false for the block before.

Reply:
No can keep the else I suppose! Good spot